docs/5.concepts/10.actions.mdx

```diff
@@ -74,10 +74,10 @@ We can access different types of information inside the `botonicInit` function.
 - **input:** the type of data and the user input.  
   `{'type': 'text', 'data': 'Hello, it's my first interaction'}`
 - **session**: the session is a group of variables that is always available at any action of the bot. It's recalculated every time there is an user input, so if a variable is updated in any state of the bot, this variable is going to be available and updated in all the others actions.
-- **params**: when a [Button](/main-concepts/components) redirects you to an action, you can pass attributes (parameters) to that action. The params store this information.
+- **params**: when a button redirects you to an action, you can pass attributes (parameters) to that action. The params store this information.
 - **lastRoutePath**: the last path that the user has gone through.
 
-The `render()` method describes what will be returned as an output of the bot using [Components](/api/components).
+The `render()` method describes what will be returned as an output of the bot using components.
```

Comment:
instead of "describes", maybe "implements", "defines" or "processes" are better words?

Reply:
I used "defines"